README.md

```diff
@@ -43,12 +43,16 @@ then the workspace looks like below
 ```
 $ cd <workspace>/build_media
 ```
-6. 
+6.  Export the current LIBVA installation directory(ies)  
```

Comment:
Imho, this should not be needed at all. At most this might be an optional step.

Reply:
As of now it's a just in case/can't hurt to deal with the wrong directories being chosen by default. 

Regardless I'm fine with putting `(Optional)` in front of it